Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -757,7 +762,10 @@ defineSuite([
                 expect(tileset._root.visibility(scene.frameState.cullingVolume)).not.toEqual(CullingVolume.MASK_OUTSIDE);
             });
 
+            var spyUpdate = jasmine.createSpy('listener');
+            tileset.loadProgress.addEventListener(spyUpdate);
             scene.renderForSpecs();
+            expect(spyUpdate.calls.count()).toEqual(1);
```

Comment:
Like above, remove the original `addEventListener` and make sure to check `spyUpdate.calls.argsFor`. The arg should be the root tile.